### DISCUSSION TITLE
Sound arbitrary length and latent sampling

### DISCUSSION THREAD

**Discussion by ghost** [v-iashin/SpecVQGAN#9] [Ideas]
Hello!

Congratulations in advance for your work. I create this post here to try to understand how I can use your code to generate wav files of arbitrary size (instead of 9 seconds, as is the default, if it would be somewhat interesting to ask for 2 minutes for example - being in coherence with the number of trimmed frames). As you have the drum example on the project page, is this something that would be quick for me to implement?

Yours sincerely,
Luis

**Comment by ghost** [comment]
Ok, I just realized that we have this variable -> `W_scale (int > 0)`: the output sound duration will be `W_scale * 9.8 seconds`

This will be my wav file duration right? When specifying the sampling parameters.

**Reply by v-iashin** [reply]
Yes, I think you got it right, and thank you!
